### DISCUSSION TITLE
My github repository doesn't recognize .ipynb

### DISCUSSION THREAD

**Discussion by BOSOEK** [github-linguist/linguist#5482] [Classification] [answered]
## Describe the bug
Hello, I am a student studying artificial intelligence. I uploaded the .ipynb code to github, but from one day github did not recognize any language  
I deleted all the files on github and uploaded them again, and I also made a .gitattributes file, but none of them worked.
This is my repository -> https://github.com/BOSOEK/MS_Pytorch

Please help me!

**Comment by lildude** [comment, marked as answer]
To quote what I said at https://github.com/github/linguist/discussions/5456#discussioncomment-1045057:

> That said, your repo isn't analysed for the same reason as that discussed in #5445 . From my [#5445 (comment)](https://github.com/github/linguist/discussions/5445#discussioncomment-987928)
> 
> > The issue here is that repo is heeeeaaawg and has a massive object tree and a lot of files so it hitting the guard at:
> > https://github.com/github/linguist/blob/16c70aef8cd62ca071231a380c69050f5e83c900/lib/linguist/repository.rb#L132-L135

The biggest culprit are all the `.txt` files in https://github.com/BOSOEK/MS_Pytorch/tree/main/Nlp/.data/imdb/aclImdb/train/unsup - there are 50000 files in this directory alone 😱.

The only way you can get the analysis to be performed is by reducing the number of files in the repo and that directory is probably a good place to start.

As this is not a bug, I'm moving this to Discussions.

**Reply by BOSOEK** [reply]
Thank you for your help. 
After removing all data files and committing again, the language display was displayed. 
very very thank you!
